### DISCUSSION TITLE
QSO confirmation not currently displayed on QSO detail lightbox

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#857] [General]
When clicking on a logbook entry, would it be possible to show any eQSL/LoTW confirmation details in the QSO lightbox?

**Comment by AndreasK79** [comment]
You mean more detail than this?
![image](https://user-images.githubusercontent.com/6977712/105956614-5ac2f400-6078-11eb-9544-143d512931b4.png)

However I did look at what data can be displayed here, and it seems that eQSL is not displayed.

**Comment by magicbug** [comment]
As @AndreasK79 pointed out LoTW already does, I don't think eQSL does, that said the popup probably needs cleaned up a bit.

**Comment by ofadam** [comment]
I've cleaned things up and added eQSL – a pull request has been submitted.

**Comment by magicbug** [comment]
This has now been implimented
